.gitignore

```diff
@@ -0,0 +1,3 @@
+/bin

```

Comment:
Does this work? I usually see stuff like `bin/` but if this works that's fine

Reply:
Apparently it does not, oops

Reply:
Heh :)